utils.py

```diff
@@ -71,3 +71,7 @@ def setup_checkpoints(model, optimizer):
         max_to_keep=1
     )
     return checkpoint, checkpoint_manager
+
+
+def normalise_by_max(spectrogram):
+    return spectrogram / np.max(np.abs(spectrogram))
```

Comment:
Is it ok to normalize spectrograms ? Shouldn't we normalize audio files instead ?

Reply:
Why would it not be ok ?

Reply:
I've realized that normalizing FFT is probably the same as normalizing song amplitudes anyway. So let's keep it like that.